parser/src/matrix.rs

```diff
@@ -1,7 +1,7 @@
 //!
 use std::ops::{Add, AddAssign, Mul, Sub};
 
-use crate::prelude::*;
+use crate::{ast::MulType, prelude::*};
```

Comment:
move to prelude as above?
```suggestion
use crate::prelude::*;
```